content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -415,6 +415,17 @@ approve node _serving_ certificates for [security
 reasons](https://github.com/kubernetes/community/pull/1982). To use
 `RotateKubeletServerCertificate` operators need to run a custom approving
 controller, or manually approve the serving certificate requests.
+
+A deployment-specific approver for kubelet serving certificates should typically only approve CSRs which:
```

Comment:
- Is "A deployment-specific approver" referring to a custom controller? What do you think about adding "controller".

Reply:
it's philosophically a controller, but could be a manual process (e.g. "run `kubectl describe csr/$name`, review, then run `kubectl certificate approve $name`")

Reply:
OK. "approver" sounds vague. Perhaps this is previously defined in the page.

Reply:
would changing it to "A deployment-specific approval process for kubelet serving certificates..." be better?

Reply:
Sure. Thanks!

Reply:
updated